### PR TITLE
implement tnfw profile

### PR DESCRIPTION
**Pull request by dangilman**
figured it out

### CODE REVIEW

**Comment by dangilman** [comment]
as in I figured out the potential

File: lenstronomy/LensModel/Profiles/tnfw.py [hunk 1]
```diff
@@ -323,95 +322,59 @@ def _g(self, x, tau):
...
+
+        if isinstance(x, np.ndarray) or isinstance(x, list):
+            out = np.empty_like(x)
+            inds1 = np.where(x < 1)
+            inds2 = np.where(x >= 1)
+
+            out[inds1] = -np.arccosh(1 / x[inds1]) ** 2
+            out[inds2] = np.arccos(1 / x[inds2]) ** 2
+
+        elif isinstance(x, float) or isinstance(x, int):
```
Comment (sibirrer) [inline lenstronomy/LensModel/Profiles/tnfw.py]: this part is not tested

File: lenstronomy/LensModel/Profiles/tnfw.py [hunk 2]
```diff
@@ -323,95 +322,59 @@ def _g(self, x, tau):
...
+            if x < 1:
+                out = -np.arccosh(1 / x) ** 2
+            else:
+                out = np.arccos(1 / x) ** 2
+
```
Comment (sibirrer) [inline lenstronomy/LensModel/Profiles/tnfw.py]: should there be a raise statement when none of the two options are valid as the variable 'out' is not defined?

**Comment by dangilman** [comment]
Done

**Pull request merged**